pycode/memilio-surrogatemodel/memilio/surrogatemodel/ode_secir_simple/model.py

```diff
@@ -0,0 +1,190 @@
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#############################################################################
+import numpy as np
+import pandas as pd
+import os
+import pickle
+import tensorflow as tf
+import matplotlib.pyplot as plt
+from memilio.surrogatemodel.ode_secir_simple import data_generation
+from memilio.surrogatemodel.ode_secir_simple import different_networks
+
+
+def plotCol(
+        inputs, labels, model=None, plot_col='InfectedSymptoms',
+        max_subplots=8):
+    """! Plot prediction of the model and label for one compartment.
+
+    @param inputs test inputs for model prediction. 
+    @param labels test labels. 
+    @param model trained model. 
+    @param plot_col string name of compartment to be plotted. 
+    @param max_subplots number of plot to be plotted. 
+    """
+
+    input_width = inputs.shape[1]
+    label_width = labels.shape[1]
+
+    plt.figure(figsize=(12, 8))
+    cols = np.array([
+        'Susceptible', 'Exposed', 'InfectedNoSymptoms', 'InfectedSymptoms',
+        'InfectedSevere', 'InfectedCritical', 'Recovered', 'Dead'])
+    plot_col_index = np.where(cols == plot_col)[0][0]
+    max_n = min(max_subplots, inputs.shape[0])
+
+    for n in range(max_n):
+        plt.subplot(max_n, 1, n+1)
+        plt.ylabel(f'{plot_col}')
+
+        input_array = inputs[n].numpy()
+        label_array = labels[n].numpy()
+        plt.plot(np.arange(0, input_width), input_array[:, plot_col_index],
+                 label='Inputs', marker='.', zorder=-10)
+        plt.scatter(
+            np.arange(input_width, input_width + label_width),
+            label_array[:, plot_col_index],
+            edgecolors='k', label='Labels', c='#2ca02c', s=64)
+
+        if model is not None:
+            input_series = tf.expand_dims(inputs[n], axis=0)
+            pred = model(input_series)
+            pred = pred.numpy()
+            plt.scatter(np.arange(input_width, input_width+pred.shape[-2]),
+                        pred[0, :, plot_col_index],
+                        marker='X', edgecolors='k', label='Predictions',
+                        c='#ff7f0e', s=64)
+
+    plt.xlabel('days')
+    if os.path.isdir("plots") == False:
+        os.mkdir("plots")
+    plt.savefig('plots/evaluation_secir_simple_' + plot_col + '.pdf')
+
+
+def network_fit(path, model, max_epochs=30, early_stop=500, plot=True):
+    """! Training and evaluation of the model. 
+
+    @param path path of the dataset. 
+    @param model name of the model to be loaded from the file where the model architectures are saved. 
```

Comment:
```suggestion
    @param model Keras sequential model.
```

It's not the name, is it? (Dont know if it needs to be sequential though)